### PR TITLE
fix(icons): redesigned `house` icon

### PR DESCRIPTION
**Pull request by jguddas**
## What is the purpose of this pull request?
- [x] Other: Icon update

### Description
Redesigned `home` icon

## Before Submitting
- [x] I've read the [Contribution Guidelines](https://github.com/lucide-icons/lucide/blob/main/CONTRIBUTING.md).
- [x] I've checked if there was an existing PR that solves the same issue.

### CODE REVIEW

File: icons/home.svg
Comment (karsa-mistmere) [inline icons/home.svg:1]: This one looks weird when scaled up:
image

Comment (karsa-mistmere) [reply icons/home.svg:1]: There's also some weird artefact, at the bottom left corner:
image

**Comment by jguddas** [comment]
What makes the curved door feel weird to you @karsa-mistmere?
Looks pretty neat at 12x to me.
Frame

File: icons/home.svg [hunk 1]
```diff
@@ -9,6 +9,6 @@
   stroke-linecap="round"
   stroke-linejoin="round"
 >
-  <path d="m3 9 9-7 9 7v11a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2z" />
-  <polyline points="9 22 9 12 15 12 15 22" />
+  <path d="M3 19A2 2 0 0 0 5 21h14a2 2 0 0 0 2-2v-9a2 2 0 0 0-.772-1.579L12 2 3.772 8.421A2 2 0 0 0 3 10z" />
+  <rect x="9" y="12" width="6" height="9" rx="1" />
```
Comment (karsa-mistmere) [inline icons/home.svg:13]: Shouldn't the top corner also be rounded?

```suggestion
  <path d="M15 21v-8a1 1 0 0 0-1-1h-4a1 1 0 0 0-1 1v8" />
  <path d="M3 9.923a2 2 0 0 1 .709-1.451l7-5.999a2 2 0 0 1 2.582 0l7 5.999A2 2 0 0 1 21 10.001V19a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2.001z" />
```

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M15+21v-8a1+1+0+0+0-1-1h-4a1+1+0+0+0-1+1v8%22+/%3E%3Cpath+d=%22M3+9.923a2+2+0+0+1+.709-1.451l7-5.999a2+2+0+0+1+2.582+0l7+5.999A2+2+0+0+1+21+10.001V19a2+2+0+0+1-2+2H5a2+2+0+0+1-2-2.001z%22+/%3E&name=image">icons<br/>Open lucide studio</a>

image

**Review by karsa-mistmere — CHANGES_REQUESTED** [review]

**Comment by karsa-mistmere** [comment]
Also up for discussion: should we rename this to `house`?

**Comment by karsa-mistmere** [comment]
> What makes the curved door feel weird to you @karsa-mistmere? Looks pretty neat at 12x to me. Frame

To me that looks awful, I very much prefer this:

image

**Comment by ericfennis** [comment]
Can we also take a look at #1841 before we merge this one.

Other related issues:
- #2098 
- #1855

**Comment by karsa-mistmere** [comment]
> Can we also take a look at #1841 before we merge this one.
> 
> Other related issues:
> 
> * [[Icon Request] "Home Plus" icon #2098](https://github.com/lucide-icons/lucide/issues/2098)
> * [[Icon Request]: Energy Autarky for house #1855](https://github.com/lucide-icons/lucide/issues/1855)

@jguddas already designed this very cute icon at #2123 for the autarky use case

As for `home-plus` (or should it not be `house-plus`?), I think these are our best candidates:
image

None of them spark joy, but it should definitely not be the last one.

**Comment by jguddas** [comment]
> > Can we also take a look at #1841 before we merge this one.
> > Other related issues:
> > 
> > * [[Icon Request] "Home Plus" icon #2098](https://github.com/lucide-icons/lucide/issues/2098)
> > * [[Icon Request]: Energy Autarky for house #1855](https://github.com/lucide-icons/lucide/issues/1855)
> 
> @jguddas already designed this very cute icon at #2123 for the autarky use case
> 
> As for `home-plus` (or should it not be `house-plus`?), I think these are our best candidates: image
> 
> None of them spark joy, but it should definitely not be the last one.

The second one with the plus in the roof looks nice.

**Comment by ericfennis** [comment]
@karsa-mistmere @jguddas I vote as well for the second one!

**Comment by karsa-mistmere** [comment]
> Also up for discussion: should we rename this to house?

This still bugs me, I feel like `house` is what's depicted and `home` is a use case.

**Review by karsa-mistmere — APPROVED** [review]

**Comment by karsa-mistmere** [comment]
> > Also up for discussion: should we rename this to house?
> 
> This still bugs me, I feel like `house` is what's depicted and `home` is a use case, so as per naming convention no. 4, this should be `house`, not `home`.

@ericfennis & @jguddas: still waiting for your two cents on this.

---

Here's the finalized `house-plus`/`home-plus` icon:

<a title="Open lucide studio" target="_blank" href="https://lucide-studio.vercel.app/edit?value=%3Cpath+d=%22M13.22+2.416a2+2+0+0+0-2.511.057l-7+5.999A2+2+0+0+0+3+10v9a2+2+0+0+0+2+2h14a2+2+0+0+0+2-2v-7.354%22+/%3E%3Cpath+d=%22M15+21v-8a1+1+0+0+0-1-1h-4a1+1+0+0+0-1+1v8%22+/%3E%3Cpath+d=%22M15+6h6%22+/%3E%3Cpath+d=%22M18+3v6%22+/%3E&name=image">icons<br/>Open lucide studio</a>

**Comment by jguddas** [comment]
I vote for keeping it home.

**Comment by ericfennis** [comment]
@karsa-mistmere is right here. If we want to be consistent with renaming, we should rename this to `house`.

**Review by ericfennis — APPROVED** [review]

**Pull request merged**